rflx/generator/common.py

```diff
@@ -68,59 +69,67 @@ def substitution(
         self, message: Message, prefix: bool = True, public: bool = False
     ) -> Mapping[Name, Expr]:
         def prefixed(name: str) -> Expr:
-            return Selected(Name("Ctx"), name) if prefix else Name(name)
+            return Variable(f"Ctx.{name}") if prefix else Variable(name)
```

Comment:
Because I thought `Variable(f"Ctx.{name}")` is easier to understand than `Selected(Variable("Ctx"), name)`. But at the end both variants are equivalent.

Reply:
Seeing both side-by-side I don't think using `Selected` is a real advantage. Leave it as is.